src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css

```diff
@@ -23213,6 +23219,7 @@ a[dir]:root[data-theme=default] .badge-dark:focus, a[data-theme=default][dir]:ro
   cursor: pointer;
   transition-duration: 0.2s;
   transform: scale(2, 1);
+  font-size: 0;
```

Comment:
Is this really enough to make the text invisible? Check for Bootstrap classes that we may use instead, which are safer.

Reply:
I am thinking about using `display: none` but the problem I have at the moment is that this removes even the icon too.

Can I put the button text in a `<span>.` This way, I can easily target the correct part of the button. I don't think this would go against the rule.

Reply:
@Piedone What do you think?

Reply:
Just please do whatever you think is suitable, you needn't check before trying out little things like this.